### PR TITLE
Add possibility to configure timeout for plugin broker

### PR DESCRIPTION
**Pull request by mshaposhnik**
### What does this PR do?
Add configuration for result waiting timeout in plugin broker

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/11120

#### Release Notes
N/A

#### Docs PR
N/A

### CODE REVIEW

File: assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties [hunk 1]
```diff
@@ -369,6 +369,9 @@ che.infra.kubernetes.workspace_start_timeout_min=8
 # Defines the timeout in minutes that limits the period for which Kubernetes Ingress become ready
 che.infra.kubernetes.ingress_start_timeout_min=5
 
+# Defines the timeout in minutes that limits the max period of result waiting for plugin broker.
+che.infra.kubernetes.plugin_broker_wait_timeout_min=3
```
Comment (sleshchenko) [inline assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties]: Please rename it in the way in which other plugin broker properties are named, like `che.workspace.plugin_broker.wait_timeout_min`:
https://github.com/eclipse/che/blob/d82481bff1f08383637c84cab1a36d1a65c242bf/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties#L510-L514

And put it near them

**Review by sleshchenko — APPROVED** [review]
LGTM

Please take a look my inlined comment before a merge

**Comment by mshaposhnik** [comment]
@garagatyi do we need something more than that? Some openshift specific mappings?

**Review by garagatyi — APPROVED** [review]

**Comment by garagatyi** [comment]
nothing else

**Comment by mshaposhnik** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/865/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/865//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11368
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by mshaposhnik** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/866/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/866//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11368
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by mshaposhnik** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/867/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/867//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11368
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Pull request merged**